src/app/page.tsx

```diff
@@ -162,7 +162,14 @@ export default function Page() {
           <h2 className="text-xl font-bold">Skills</h2>
           <div className="flex flex-wrap gap-1">
             {RESUME_DATA.skills.map((skill) => {
-              return <Badge key={skill}>{skill}</Badge>;
+              return (
+                <Badge
+                  key={skill}
+                  className=" print:text-primary print:underline"
```

Comment:
How about adding a prop that allows the users to optionally get this variant instead of hard-coding it?
so that people can choose between them.

Reply:
great idea!